google-assistant-webserver/hassio_gassistant.py

```diff
@@ -28,12 +28,22 @@
 class BroadcastMessage(Resource):
     def get(self):
         message = request.args.get('message', default = 'This is a test!')
-        text_query = 'broadcast "'+message+'"'
+        text_query = 'broadcast '+message+''
         display_text = assistant.assist(text_query=text_query)
         return {'status': 'OK'}
 
 api.add_resource(BroadcastMessage, '/broadcast_message')
 
+class Command(Resource):
+    def get(self):
+        message = request.args.get('message', default = 'This is a test!')
+        text_query = ''+message+''
```

Comment:
It might be nice to do away with assigning `text_query` to a new variable here, as it isn't different from `message`. Could just use `message` by itself in the next line.